share/spack/gitlab/cloud_pipelines/stacks/e4s/spack.yaml

```diff
@@ -294,12 +295,22 @@ spack:
           - magma
           - mfem
           - mpich
+          - nvhpc
```

Comment:
nvhpc doesn't build anything. it downloads, decompresses, extracts, moves, tarballs and compresses, everything is single theaded.

Reply:
@haampie I don't know if I get your point here.  Are you saying don't run `nvhpc` in this resource group?

Reply:
It doesn't run faster if you throw more cores at it. Only compression might go faster then, but we don't have threaded compression right now